File: step2.py
import base64
import string
import random
def encode(s):
	return base64.b64encode(s).translate(
		string.maketrans(
			string.uppercase+string.lowercase+string.digits+"+/",
			string.digits+string.uppercase+string.lowercase+"+/"
		)
	)

def chunk(s):
	c=[]
	i=0
	while(i<len(s)):
		j=random.choice(range(5,20))
		c.append(s[i:i+j])
		i+=j
	return c

print '''<html><head>
<meta http-equiv="X-UA-Compatible" content="IE=10">
<meta charset="UTF-8">
</head>
<body><hl>boom boom</hl><script>'''
with open('s1') as f:
	s = f.read()
b=encode(s[:-1])
c=chunk(b)
m=range(0,len(c))
r=list(zip(m,c))
random.shuffle(r)
d='x=['
for i in r:
	d+='"'+i[1]+'",'
print d[:-1]+'];'
d='z='
for i in sorted(r):
	d+='x['+str(r.index(i))+']+'
print d[:-1]+';'
print '''k="0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz+/=";
function g(e){var t="";var n,r,i;var s,o,u,a;var f=0;e=e.replace(/[^A-Za-z0-9+/=]/g,"");while(f<e.length){s=k.indexOf(e.charAt(f++));o=k.indexOf(e.charAt(f++));u=k.indexOf(e.charAt(f++));a=k.indexOf(e.charAt(f++));n=s<<2|o>>4;r=(o&15)<<4|u>>2;i=(u&3)<<6|a;t=t+String.fromCharCode(n);if(u!=64){t=t+String.fromCharCode(r)}if(a!=64){t=t+String.fromCharCode(i)}}return t;}
eval(g(z));
</script>
<h5>pop pop</h5></body></html>'''

File: step2.js
var key="0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz+/="
function decode(input){
	var out="";
	var enc1;
	var enc2;
	var enc3;
	var char1;
	var char2;
	var char3;
	var char4;
	var index=0;
	input=input.replace(/[^A-Za-z0-9+/=]/g,"");
	while(index<input.length){
		char1=key.indexOf(input.charAt(index++));
		char2=key.indexOf(input.charAt(index++));
		char3=key.indexOf(input.charAt(index++));
		char4=key.indexOf(input.charAt(index++));
		enc1=char1<<2|char2>>4;
		enc2=(char2&15)<<4|char3>>2;
		enc3=(char3&3)<<6|char4;
		out=out+String.fromCharCode(enc1);
		if(enc2!=64){out=outt+String.fromCharCode(enc2)}
		if(enc3!=64){out=out+String.fromCharCode(enc3)}
	}
	return out
}
